cli/args/mod.rs

```diff
@@ -1082,7 +1081,7 @@ impl CliOptions {
             let specifier = specifier.clone();
             async move {
               let file = file_fetcher
-                .fetch(&specifier, &PermissionsContainer::allow_all())
+                .fetch_bypass_permissions(&specifier)
```

Comment:
I see where this is coming from, but for https://github.com/denoland/deno/pull/25469 this will have to be rewritten anyway. I guess we can land it for now though